.github/workflows/build-git-installers.yml

```diff
@@ -213,11 +213,11 @@ jobs:
 
           sed -i -e '6 a use_recently_seen=no' \
             $b/git-update-git-for-windows
-      - name: Set the installer Publisher to the Git Fundamentals team
+      - name: Set the installer Publisher to the GitClient team
```

Comment:
Since it's repeated, maybe it is on purpose, but should there be a space here? "Git Client"

Reply:
+1 to this.. a space in our formal, display name :)